compiler/damlc/tests/daml-test-files/ExceptionCatchError.EXPECTED.desugared-daml

```diff
@@ -0,0 +1,40 @@
+module ExceptionCatchError where
+import (implicit) qualified DA.Internal.Record
+import (implicit) qualified GHC.Types
+import (implicit) qualified DA.Internal.Desugar
+import (implicit) DA.Internal.RebindableSyntax
+import DA.Assert
+import DA.Exception
+test1
+  = scenario
+      do p <- getParty "Alice"
+         submit
+           p
+           do DA.Internal.Desugar._tryCatch
+                \ _ -> do error "hello"
+                \case
+                  DA.Internal.Desugar.fromAnyException -> DA.Internal.Desugar.Some (GeneralError msg)
+                    -> DA.Internal.Desugar.Some msg === "hello"
```

Comment:
Oh ... this is confusing. `DA.Internal.Desugar.fromAnyException -> DA.Internal.Desugar.Some (GeneralError msg)` is a view pattern! We should really start adding parentheses everywhere in our desugarings. But this is unrelated to this PR.